Description: Disabled tests that failed when run on a single CPU 
 The disabled tests are
 ./06-bridge-clean-session-csF-lcsF.py
 ./06-bridge-clean-session-csF-lcsN.py
 ./06-bridge-clean-session-csF-lcsT.py
 ./06-bridge-clean-session-csT-lcsF.py
 ./15-persist-client-expired-session.py
 ./15-persist-client-v5-0.py
Author: Joachim Zobel <jz-2017@heute-morgen.de<
Bug: https://github.com/eclipse-mosquitto/mosquitto/issues/3645
Last-Update: 2026-05-09
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/test/broker/Makefile
+++ b/test/broker/Makefile
@@ -127,10 +127,10 @@
 	./06-bridge-br2b-disconnect-qos1.py
 	./06-bridge-br2b-disconnect-qos2.py
 	./06-bridge-br2b-remapping.py
-	./06-bridge-clean-session-csF-lcsF.py
-	./06-bridge-clean-session-csF-lcsN.py
-	./06-bridge-clean-session-csF-lcsT.py
-	./06-bridge-clean-session-csT-lcsF.py
+	#./06-bridge-clean-session-csF-lcsF.py
+	#./06-bridge-clean-session-csF-lcsN.py
+	#./06-bridge-clean-session-csF-lcsT.py
+	#./06-bridge-clean-session-csT-lcsF.py
 	./06-bridge-clean-session-csT-lcsN.py
 	./06-bridge-clean-session-csT-lcsT.py
 	./06-bridge-fail-persist-resend-qos1.py
@@ -283,7 +283,6 @@
 PERSIST_TESTS = \
 	./15-persist-bridge-queue.py \
 	./15-persist-client-drop-expired-messages.py \
-	./15-persist-client-expired-session.py \
 	./15-persist-client-msg-in-v3-1-1.py \
 	./15-persist-client-msg-in-v5-0.py \
 	./15-persist-client-msg-modify-acl.py \
@@ -294,7 +293,6 @@
 	./15-persist-client-msg-out-v3-1-1.py \
 	./15-persist-client-msg-out-v5-0.py \
 	./15-persist-client-v3-1-1.py \
-	./15-persist-client-v5-0.py \
 	./15-persist-client-will.py \
 	./15-persist-publish-properties-v5-0.py \
 	./15-persist-retain-clear.py \
@@ -303,6 +301,10 @@
 	./15-persist-subscription-v3-1-1.py \
 	./15-persist-subscription-v5-0.py
 
+# Temporarily removed
+# 	./15-persist-client-expired-session.py \
+#	./15-persist-client-v5-0.py \
+
 PERSIST_PLUGINS =  \
 	persist_sqlite
 
